cmd/compute-domain-kubelet-plugin/main.go

```diff
@@ -35,8 +35,9 @@ import (
 )
 
 const (
-	DriverName                         = "compute-domain.nvidia.com"
-	DriverPluginCheckpointFileBasename = "checkpoint.json"
+	DriverName                              = "compute-domain.nvidia.com"
+	DriverPluginCheckpointFileBasename      = "checkpoint.json"
+	IMEXDaemonsWithDNSNamesMinDriverVersion = "570.158.01"
```

Comment:
I wondered where this is documented and found

> Hostnames in the nodes_config.cfg are supported, allowing the customer to change the DNS/hostname mappings at the system level without having to restart IMEX.

in https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-158-01/index.html